### PR TITLE
 Nisha | Test | Make GIDDH website compliance to UK terms and condition

### PR DESCRIPTION
**Pull request by nisha-walkover**
https://app.clickup.com/t/86curvay3

### CODE REVIEW

File: apps/web-giddh/src/app/custom-fields/list/list.component.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
     <div class="container-fluid mr-t15 custom-field-page">
         <div class="d-flex justify-content-between custom-field-header">
             <a
+                href="javascript::void(0);"
+                aria-label="add nav"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/custom-fields/list/list.component.html:13]: Remove duplicate colon

File: apps/web-giddh/src/app/custom-fields/list/list.component.ts [hunk 2]
```diff
@@ -113,6 +113,8 @@ export class CustomFieldsListComponent implements OnInit, OnDestroy {
     public deleteCustomField(customFieldUniqueName: any): void {
         let dialogRef = this.dialog.open(ConfirmModalComponent, {
             width: '500px',
+            role: 'alertdialog',
+            ariaLabel: 'Dialog ARIA label',
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/custom-fields/list/list.component.ts]: give proper label

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts [hunk 3]
```diff
@@ -114,7 +114,9 @@ export class ExportsComponent implements OnInit, OnDestroy {
     public openDialog(row: any): void {
         this.dialog.open(ExportsJsonComponent, {
             data: row?.filters,
-            panelClass: 'download-json-panel'
+            panelClass: 'download-json-panel',
+            role: 'alertdialog',
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/downloads/components/exports/exports.component.ts:118]: give proper label

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html [hunk 4]
```diff
@@ -1411,7 +1411,7 @@ <h3 class="mr-b1">Note</h3>
         <div class="form-header">
             <h3 class="aside-title">Manage Transporter</h3>
         </div>
-        <button type="button" class="mr-t05" id="close" (click)="toggleTransporterModel()">
+        <button type="button" class="mr-t05" id="closeDialog" (click)="toggleTransporterModel()">
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html]: give proper id we are not using camel case in id

File: apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.html [hunk 5]
```diff
@@ -415,7 +415,7 @@ <h2 class="mb-2">
     <div class="right-modal account-aside-menu-state">
         <div class="aside-overlay" [ngClass]="status ? 'in' : 'out'"></div>
         <div class="aside-pane aside-right" [ngClass]="status ? 'in' : 'out'">
-            <button type="button" class="btn btn-sm btn-primary" id="close" (click)="OpenTransporterModel()">
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.html:418]: give proper id we are not using camel case in id

File: apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html [hunk 6]
```diff
@@ -317,7 +317,7 @@
                                                 <a
                                                     href="javascript:void(0)"
                                                     class="dropdown-toggle"
-                                                    id="button-basic"
+                                                    id="actionButton"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html]: give proper id we are not using camel case in id

File: apps/web-giddh/src/app/invoice/preview/models/advanceSearch/invoiceAdvanceSearch.component.html [hunk 7]
```diff
@@ -1,10 +1,12 @@
...
+<div role="alertdialog"
+aria-labelledby="dialog1Title"
+aria-describedby="dialog1Desc">
+    <div class="modal-header theme-bg pd-2 pd-l2 pd-r2 clearfix" role="document" tabindex="0">
+        <h2 class="modal-title bg" id="dialog1Title">{{ commonLocaleData?.app_advance_search }}</h2>
         <i class="icon-cross text-right close-modal" aria-hidden="true" (click)="onCancel()"></i>
     </div>
 
-    <div class="modal-body pd-l2 pd-r2 clearfix" id="export-body">
+    <div class="modal-body pd-l2 pd-r2 clearfix" id="dialog1Desc">
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/preview/models/advanceSearch/invoiceAdvanceSearch.component.html]: give proper id we are not using camel case in id

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.scss [hunk 8]
```diff
@@ -1,4 +1,4 @@
-#close {
+#closeDialog {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.scss]: Correct css id

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.html [hunk 9]
```diff
@@ -3,7 +3,7 @@
         <div class="header-create-unit">
             <div class="form-header">
                 <h3 mat-dialog-title>{{ commonLocaleData?.app_create_group }}</h3>
-                <button id="close" mat-dialog-close><i class="icon-cross"></i></button>
+                <button id="closeDialog" aria-label="close button" mat-dialog-close><i class="icon-cross"></i></button>
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.html]: correct button id name

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.html [hunk 10]
```diff
@@ -3,7 +3,7 @@
         <div class="header-create-unit">
             <div class="form-header">
                 <h3 mat-dialog-title>{{ commonLocaleData?.app_create_unit }}</h3>
-                <button id="close" mat-dialog-close><i class="icon-cross"></i></button>
+                <button id="closeDialog" aria-label="close button" mat-dialog-close><i class="icon-cross"></i></button>
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.html]: Correct id name

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.scss [hunk 11]
```diff
@@ -1,4 +1,4 @@
-#close {
+#closeDialog {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.scss]: Correct id;s name in overall files where you are using camel case format

File: apps/web-giddh/src/app/new-inventory/component/branch-transfer/list-branch-transfer/list-branch-transfer.component.ts [hunk 12]
```diff
@@ -571,7 +571,9 @@ export class ListBranchTransferComponent implements OnInit {
         this.voucherTypeDropdown?.closeDropdownPanel();
         this.dialog.open(this.advanceFilterComponent, {
             width: '500px',
-            autoFocus: false
+            autoFocus: false,
+            role: 'alertdialog',
+            ariaLabel: 'Dialog ARIA label',
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/branch-transfer/list-branch-transfer/list-branch-transfer.component.ts]: Correct label name

File: apps/web-giddh/src/app/new-inventory/component/create-unit-group/create-unit-group.component.ts [hunk 13]
```diff
@@ -145,6 +145,8 @@ export class CreateUnitGroupComponent implements OnInit, OnChanges, OnDestroy {
     public deleteUnitGroup(): void {
         let dialogRef = this.dialog.open(ConfirmModalComponent, {
             width: '40%',
+            role: 'alertdialog',
+            ariaLabel: 'Dialog ARIA label',
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/create-unit-group/create-unit-group.component.ts]: Correct dialog aril label name in overall changes

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html [hunk 14]
```diff
@@ -116,6 +116,7 @@ <h4 class="clearfix bd-rb">
                                             name="sendThroughGmail"
                                             color="primary"
                                             [disabled]="!isGmailIntegrated"
+                                            aria-label="sendThroughGmail"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html:119]: Correct aria label name in over all changes

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html [hunk 15]
```diff
@@ -423,12 +425,13 @@ <h4 class="modal-title bg pull-left">{{ localeData?.page_heading }}</h4>
                                     </div>
 
                                     <div class="form-group row clearfix">
-                                        <label class="col-md-2"
+                                        <label for="calculatedTaxAmount" class="col-md-2"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html:428]: Correct name also her input for

File: apps/web-giddh/src/app/voucher/voucher.component.html [hunk 16]
```diff
@@ -1155,12 +1175,15 @@ <h1>{{ invoiceTypeLabel }}</h1>
...
-                                                purchaseBillCompany.shippingDetails?.state?.name ||
-                                                purchaseBillCompany.shippingDetails?.state?.code ||
-                                                purchaseBillCompany.shippingDetails?.stateCode
+                                                (purchaseBillCompany.shippingDetails?.gstNumber ||
+                                                    purchaseBillCompany.shippingDetails?.taxNumber ||
+                                                    purchaseBillCompany.shippingDetails?.stateName ||
+                                                    purchaseBillCompany.shippingDetails?.state?.name ||
+                                                    purchaseBillCompany.shippingDetails?.state?.code ||
+                                                    purchaseBillCompany.shippingDetails?.county?.name ||
+                                                    purchaseBillCompany.shippingDetails?.county?.code ||
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/voucher/voucher.component.html:1184]: Check this code in test branch

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 17]
```diff
@@ -1764,7 +1768,7 @@ input {
         .profile-tab-container {
             .account-aside-menu-state {
                 .aside-pane {
-                    #close {
+                    #closeDialog {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/assets/css/style-2.scss]: correct id's name

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 18]
```diff
@@ -4480,18 +4485,18 @@ select.form-control {
...
-.grp-list li,
-.new-btn-ul li {
+.grp-list .master-list,
+.new-btn-ul .new-btn {
     list-style: none;
 }
 
-.grp-list li a,
-.new-btn-ul li a {
+.grp-list .master-list a,
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/assets/css/style-2.scss:4493]: correct format in scss

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/custom-fields/list/list.component.html (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/custom-fields/list/list.component.html:13]: done

File: apps/web-giddh/src/app/custom-fields/list/list.component.ts (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/custom-fields/list/list.component.ts]: done

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts (hunk 3 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/downloads/components/exports/exports.component.ts:118]: done

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html (hunk 4 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html]: done

File: apps/web-giddh/src/app/downloads/components/exports/exports.component.ts (hunk 3 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/downloads/components/exports/exports.component.ts:118]: done

File: apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.html (hunk 5 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.html:418]: done

File: apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html (hunk 6 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html]: done

File: apps/web-giddh/src/app/invoice/preview/models/advanceSearch/invoiceAdvanceSearch.component.html (hunk 7 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/invoice/preview/models/advanceSearch/invoiceAdvanceSearch.component.html]: done

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.scss (hunk 8 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.scss]: done

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.html (hunk 9 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/aside-create-unit-group/aside-create-unit-group.component.html]: done

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.html (hunk 10 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.html]: done

File: apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.scss (hunk 11 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/aside-create-unit/aside-create-unit.component.scss]: done

File: apps/web-giddh/src/app/new-inventory/component/branch-transfer/list-branch-transfer/list-branch-transfer.component.ts (hunk 12 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/branch-transfer/list-branch-transfer/list-branch-transfer.component.ts]: done

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html (hunk 14 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html:119]: done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 17 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: done

File: apps/web-giddh/src/app/voucher/voucher.component.html (hunk 16 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/voucher/voucher.component.html:1184]: done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**